docs/sandbox-permissions.rst

```diff
@@ -1,40 +1,94 @@
+
+Interface toolkits like GTK and Qt implement transparent support for portals, meaning that applications don't need to any additional work to use them (it is worth checking which portals each toolkit supports). Applications that aren't using a toolkit with support for portals can refer to the `xdg-desktop-portal API documentation <https://flatpak.github.io/xdg-desktop-portal/portal-docs.html>`_ for information on how to use them.
+
+Permissions guidelines
+----------------------
+
+While application developers have control over the sandbox permissions they wish to configure, good practice is encouraged and can be enforced. For example, the Flathub hosting service places requirements on which permissions can be used, and software on the host may warn users if certain permissions are used.
+
+The following guidelines describe which some permissions can be freely used, which can be used on an as-needed basis, and which should be avoided.
+
+Standard permissions
+````````````````````
+
+The following permissions provide access to basic resources that applications commonly require, and can therefore be freely used:
+
+- ``--share=network`` - access the network
+- ``--socket=x11`` - show windows using X11
+- ``--share=ipc`` - share IPC namespace with the host (necessary for X11)
+- ``--socket=wayland`` - show windows with Wayland
+- ``--device=dri`` - OpenGL rendering
+- ``--socket=pulseaudio`` - play sound with PulseAudio
+
+D-Bus access
+````````````
+
+Applications are automatically granted access to their own namespace. Access beyond this is typically unnecessary, although there are a small number of exceptions, such as using `MPRIS to provide media controls <https://www.freedesktop.org/wiki/Specifications/mpris-spec/>`_.
+
+Access to the entire bus with ``--socket=system-bus`` or ``--socket=session-bus`` should be avoided, unless the application is a development tool. Talk permissions can be freely used, although it is recommended to use the minumum required.
+
+Filesystem access
+`````````````````
+
+It is common for applications to require access to different parts of the host filesystem, and Flatpak provides a flexible set of options for this. Some examples include:
+
+- ``--filesystem=host`` - access all files on the host
+- ``--filesystem=home`` - access the user's home directory
+- ``--filesystem=/path/path`` - access specific paths
+- ``--filesystem=xdg-download`` - access a specific XDG folder
+
+The full list the available filesystem options can be found in the :doc:`sandbox-permissions-reference`.
+
+As a general rule, Filesystem access should be limited as much as possible. home directory access is preferable to host access, XDG directory access is preferable to home directory access, and so on. Read-only access should be used wherever possible, using the ``:ro`` option. Portals should be considered as an alternative to blanket filesystem access wherever possible.
```

Comment:
> home directory access is preferable to host access

I'm not sure that is meaningfully true. Once you have `home` access you escaped the sandbox, `host` access at that point isn't really worse and can be convenient for like mounted drives for example.

Reply:
I've had a go at fixing this.